_data/partnership/municipal/city-of-sault-ste-marie-on.yml

```
@@ -0,0 +1,33 @@
+---
+schemaVersion: '1.0'
+adminCode: city-of-sault-ste-marie-on
+projects:
+  - contact:
+      email: d.gowans@cityssm.on.ca
+      name: Dan Gowans
+    category: '43231500'
+    date:
+      closed: '2020-04-30'
+      started: '2020-01-08'
+      metadaLastUpdated: '2020-01-08'
```

This is causing the build to fail because of a typo in metadataLastUpdated.

projects.0.date.metadaLastUpdated: Unexpected element
projects.0.date.metadataLastUpdated: Required field missing

---

This is probably a bug with the Partnership form

---

Fixes the typo now that @smellems PR has landed
```suggestion
      metadataLastUpdated: '2020-01-08'
```